src/routes/FavoritesPage/FavoritesPage.js

```diff
@@ -1,11 +1,67 @@
 import React, { Component } from 'react';
+import PetList from '../../components/PetList';
+import PetsService from '../../services/petsService';
+import { Section } from '../../components/Utils/Utils';
+import UsersService from '../../services/usersService';
+import AuthService from '../../services/authService';
+
+export default class FavoritesPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      pets: null,
+      typeOfAnimal: [],
+      breed: [],
+      sex: '',
+      minAge: '',
+      maxAge: '',
+      more: []
+    };
+  }
+
+  componentDidMount() {
+    PetsService.getPets({})
+      .then(pets => {
+        UsersService.getUserFavorites(AuthService.getUserIDFromToken())
+          .then(favorites => {
+            const favoritePets = (pets.filter(elem => {
+              return favorites.includes(elem.PetID);
+            }));
+            this.setState({ pets: favoritePets })
+          })
+          
+      })
+      .catch(error => console.log(error));
+  }
 
-export default class FavoritesPage extends Component {  
   render() {
+    const {
+      pets,
+      typeOfAnimal,
+      breed,
+      sex,
+      minAge,
+      maxAge,
+      more
+    } = this.state;
+
     return (
-      <>
+      <Section>
         <h2>Favorites</h2>
-      </>
+        {pets && <PetList
+          pets={pets}
+          page='favorites'
+          typeOfAnimal={typeOfAnimal}
+          breed={breed}
+          sex={sex}
+          minAge={minAge}
+          maxAge={maxAge}
+          more={more}
+          inputChangeHandler={() => { }}
```

Comment:
could you possibly refresh the page when a user uncheck the heart? Right now, if I dont reload the page, the unchecked pets stay on the page.
![image](https://user-images.githubusercontent.com/71732016/156068771-dfd8cf5c-5f9d-4895-94ed-8d3622552c01.png)

Reply:
I kind of see this as a design decision. It may be nice for the user to quickly re-favorite an animal if they accidentally un-favorited it. It also allows them to mass-unfavorite a bunch of animal without waiting for the refresh each time.

I do see the benefit though. May be confusing to the user to unfavorite an animal and still see it on their favorites. What do you think?

Reply:
ah okay, I personally like it if it automatically refresh every click (also because Pets page shows both unchecked and checked pets) but I see that it can be helpful for a user. I can go either way. @asching7108 what do you think?